### PR TITLE
Adds more fields to component view (v2)

### PR DESCRIPTION
**Pull request by johnclary**
This is a copy of [this PR](https://github.com/cityofaustin/atd-moped/pull/1242), which I accidentally closed by deleting its base branch 🤦 

- https://github.com/cityofaustin/atd-data-tech/issues/15349

### Steps for testing

Using an editor and admin login locally:
1. Test that the project list loads. Toggle on the **Status update** and **Partners** fields. 
2. Add a status update to a project, then confirm the status update appears in the **Status update** column of the project list.
3. Add a component to a project that you know is within any council district. Use your SQL client to inspect that the `component_arcgis_online_view.is_within_city_limits` column is `TRUE`. 
4. ☝️ Edit the component by moving its features to be outside of any council district. Verify that `is_within_city_limits` changes to `FALSE`.
5. Create a line-based component (e.g., Bike Lane) that is roughly a mile long. Use Google Maps (or what have you) to measure the distance of the component you created. Use your SQL client to verify the distances matches the value of `length_miles_total` in the DB.

### CODE REVIEW

**Review by chiaberry — APPROVED** [review]
I gave a check mark last time, I'm giving a check mark this time!

File: moped-editor/src/queries/signals.js
Comment (frankhereford) [inline moped-editor/src/queries/signals.js:1]: 👋 I looked, and I didn't see where this was imported anywhere. Was this just hanging around? 

It looks like it's good to go, I don't get any matches for this pattern in the repo: `import.*queries.*signal`

**Review by frankhereford — APPROVED** [review]
Initially, I pulled down a copy of production's data, but the metadata in this couldn't be applied. Once I merged in `main` into this branch, it was good-to-go though. ✅ 

I've run through the testing steps using both account types, and this is all 💯 🚢 . Code looks fantastic. Thank you John!

As a totally silly aside, I almost got one mile on the nose. 19 feet off. 📏 
image

File: moped-editor/src/queries/signals.js
Comment (chiaberry) [reply moped-editor/src/queries/signals.js:1]: It used to be in the signals dashboard I believe, and I must have missed it when I was removing that dashboard

Comment (johnclary) [reply moped-editor/src/queries/signals.js:1]: yep—sorry for not adding that context! 🚮

**Comment by johnclary** [comment]
thanks, y'all! i'm bringing this into main 🚢

**Pull request merged**